### ISSUE TITLE
Model inconsistency for `Post/PatchCapellaModel` and `CapellaModel`

### ISSUE THREAD

**Issue by dominik003** [DSD-DBS/capella-collab-manager#641]
Both `PostCapellaModel` and `PatchCapellaModel` have an optional field for the description which suggests that the description does not need to be included. However, for our `CapellaModel` the description field is not optional and therefore providing an empty description when creating/updating a capella model results in a pydantic error.

**Comment by MoritzWeber0** [comment]
Shouldn't be the description always set and if there is no description, the description is an empty string?
It makes sense that the description is optional in `PatchCapellaModel` - that is how patch routes work.

**Comment by dominik003** [comment]
Yes, that was the purpose of this issue, as it was not currently the case that the description was set to an empty string if it was not specified. But I included it in the PR I mentioned, since it occurred there for me as well, and it's only a one-line correction.
